src/plugins/safeSnap/constants.ts

```diff
@@ -386,3 +386,37 @@ export const MULTI_SEND_VERSIONS: Record<
   [MULTI_SEND_VERSION.V1_2_0]: MULTI_SEND_V1_2_0,
   [MULTI_SEND_VERSION.V1_3_0]: MULTI_SEND_V1_3_0
 };
+
+// to potnetially cut down on event ranges we query, hard code some deploy blocks for contracts
+export type ContractData = {
+  network: string;
```

Comment:
more robust constants to describe contracts. unfortunately getting deploy blocks are hard to do without hard coding it